### PR TITLE
Tech debt webpack

### LINKED ISSUE #182: Provide support to build with webpack
- [x] Provides default webpack configs
- [x] Build distribution folder (should copy all required static assets including required html, bundles, and images) 
- [x] Symlinks to node_modules is not required
- [x] index.html in source directory is not modified during build

### PR DESCRIPTION
**Pull request by ghost**
Blocks issues

- https://github.com/TAMULib/DirectoryUI/issues/259
- https://github.com/TAMULib/CAP/issues/173
- https://github.com/TAMULib/SAGE/issues/422
- https://github.com/TAMULib/GIFMButtonUI/issues/48
- https://github.com/TAMULib/MagpieUI/issues/202
- https://github.com/TAMULib/SubjectLibrarianUI/issues/74
- https://github.com/TAMULib/LibraryServiceStatusSystemUI/issues/212
- https://github.com/TAMULib/ProjectManagementUI/issues/118
- https://github.com/TAMULib/MyLibraryUI/issues/318

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wvr/core",
-  "version": "2.2.2-rc.2",
+  "version": "2.2.2-rc.6",
```
Comment (kaladay) [inline package.json]: Bump this to `2.2.2-rc.8`? (or to whatever the latest version is by the time this PR is ready to go.)

**Review by kaladay — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ghost) [reply package.json]: Thanks. Will check.

Comment (ghost) [reply package.json]: https://www.npmjs.com/package/@wvr/core latest published is still 2.2.2-rc.7

Comment (ghost) [reply package.json]: Required version: https://github.com/TAMULib/MyLibraryUI/pull/319/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

Comment (ghost) [reply package.json]: Will be 2.2.2-rc.8 for development. Not yet published. Please see [weaver-webpack-upgrade](https://github.com/TAMULib/MyLibraryUI/pull/319)

**Review by jeremythuff — DISMISSED** [review]

**Review by kaladay — DISMISSED** [review]

**Review by jeremythuff — APPROVED** [review]

**Review by kaladay — APPROVED** [review]

**Pull request merged**